web/handlers.go

```diff
@@ -26,13 +26,22 @@ func (w *Web) NewHandler(h func(*Context, http.ResponseWriter, *http.Request)) h
 }
 
 func (w *Web) NewStaticHandler(h func(*Context, http.ResponseWriter, *http.Request)) http.Handler {
+	// Determine the CSP SHA directive needed for subpath support, if any. This value is fixed
+	// on server start and intentionally requires a restart to take effect.
+	app := app.New(
```

Comment:
Can we pass in the config this needs instead?

Reply:
Looks like my original approach was convoluted: I can just get it off of `Web` instead.